subsys/bluetooth/mesh/Kconfig

```diff
@@ -1227,6 +1227,20 @@ config BT_MESH_SOL_PDU_RPL_CLI_TIMEOUT
 	  for a response message to arrive. This value can be changed at runtime
 	  using @ref bt_mesh_sol_pdu_rpl_cli_timeout_set.
 
+config BT_MESH_BRG_CFG_SRV
+	bool "Support for Bridge Configuration Server model"
```

Comment:
```suggestion
	bool "Support for Subnetwork Bridge Configuration Server model"
```

Reply:
Will keep the name from the Mesh Protocol spec.